fake_joint_trajectory_controller/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package>
+  <name>fake_joint_trajectory_controller</name>
+  <version>0.7.2</version>
+  <description>Fake joint trajectory controller to run MoveIt without robot.</description>
+  <author email="isucan@google.com">Ioan Sucan</author>
+  <author email="robot.moveit@gmail.com">Sachin Chitta</author>
```

Comment:
The copyright in the script says it's written by Kei Okada.
Do we have an email-address of him and shouldn't we list him as author?
If not, then maybe you should add your name there too?
At least it's not written by Ioan and Sachin.